### PR TITLE
Bje/feature/initial architecture

### LINKED ISSUE #4: Initial Architecture
** Description **
We should establish some initial architecture so that we can build out features without many merge conflicts. 

** Planning **
We haven't agreed on anything but a super basic modular react tree should work. 

** Related Features **
Literally all of them.

### PR DESCRIPTION
**Pull request by deadbelly**
### What does this PR do? 
This branch builds out an initial directory architecture. It removes files from the boilerplate we don't anticipate needing, and adds files and directories we expect to need while building the first iteration of the app. 

### Is this a feature or Fix/Refactor?  
Feature.

### Problems Encountered & Resulting Solutions  
This was really straight forward. There are some different ways of structuring the app and I'm open to choosing a different path and making some changes. 

### Where should the reviewer start?
Pull down the branch, look at the directory structure and compare it to our planning docs.

### This PR is related to which issues:
Closes #4

### CODE REVIEW

**Review by thaomonster — APPROVED** [review]
Thanks for setting up the basic. I do like the structure of the components.

File: src/Components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+class App extends React.Component {
```
Comment (thaomonster) [inline src/Components/App/App.js]: After learning hooks today can we change app to a function.

**Review by thaomonster — CHANGES_REQUESTED** [review]

**Review by mainlyetcetera — CHANGES_REQUESTED** [review]
I think we should use the hook `useState` to make our app more performant. A big piece of PWA's is performance, so anything we can do towards that aim would help!

File: src/Components/App/App.js (hunk 1 above)
Comment (deadbelly) [reply src/Components/App/App.js]: Converted this to a component and imported `useState`.

**Review by mainlyetcetera — APPROVED** [review]
I think the change to a functional component for `App` is very appropriate. Good work!

**Review by thaomonster — APPROVED** [review]

**Pull request merged**